__tests__/6-get-api-reviews-comments.test.js

```diff
@@ -0,0 +1,59 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed")
+const connection = require("../db/connection");
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => {
+    return seed(testData);
+})
+
+afterAll(() => {
+    return connection.end()
+})
+
+describe('testing the get api/reviews/ endpoint', () => {
+    test('testing that get api/reviews/ returns the right information and returns status 200', () => {
+        return request(app)
+        .get("/api/reviews/3/comments")
+        .expect(200)
+        .then((result) => {     
+            const comment = result.body.comments
+            comment.forEach((comment) => {
+            expect(comment).toEqual(expect.objectContaining({
+                comment_id: expect.any(Number),
+                review_id: expect.any(Number),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                author: expect.any(String),
+                body: expect.any(String)         
+            }))
+        })
+    })
+    })
+    test('that the comments are returned in the correct order' , () => {
+        return request(app)
+        .get("/api/reviews/3/comments")
+        .expect(200)
+        .then((result) => {
+            const comments = result.body.comments
+            expect(comments).toBeSortedBy('created_at')
+        })
+    })
+    test('throw an error if review id does not have any information', () => {
+        return request(app)
```

Comment:
This test description isn't very semantic - "responds with a status 404 for a non existent ID.